.gitignore

```diff
@@ -42,6 +42,7 @@ out/
 ### OpenAPI CodeGen detritus
 .openapi-generator/
 pom.xml
+src/main/resources/api/cloud_resources_uid
```

Comment:
This seems strange? I'd hope that gradle doesn't materialize our yaml into src/main. (something under `build` would be better). Or was this for dev while you were getting packaging fixed?

Reply:
ah, that is the left over. Previously my build.gradle will unzip CloudResourceSchema package and copied it here then delete it. Now seems we don't need this, so that is the leftover.
On the other hand, if I unzip this under `build`, my service_openapi.yaml can not find that file, so I put this under src/

Reply:
We should fix the gradle so that it can find the file. If we tell the gradle openApiGenerate rule that it depends on the resource files from `cloud-resource-schema`, we should be able to do do that rather than unzipping into src main. Not sure exactly what the gradle looks like, but let's try to get there. Let me know if you want to chat more about what this looks like.